### ISSUE TITLE
Difference between master and gh-pages

### ISSUE THREAD

**Issue by hyperknot** [cmulders/Leaflet.Zoomify#3]
Hi, can you explain what are the differences between the two brances (only in the js files).

Master is said to be compatible with Leaflet 1.0-dev and is the one in the turban repo's PR. What is the gh-pages one for? In my experience the gh-pages one actually seems to be better with Leaflet 1.0, is that normal?

**Comment by cmulders** [comment]
Yes there are indeed two branches. My intention was to keep a clean reference to Turban's original code. However, the gh-pages branch offers some hosting to display an example, so switched to that later. Maybe now a good time to delete that branch and maybe refactor some of the code again for 1.0-rc.

**Comment by hyperknot** [comment]
@cmulders so which one would you consider the better starting point for a rc1 compatible version? 

Shouldn't it ideally be similar to L.TileLayer or L.TileLayer.WMS as a minimal example?
https://github.com/Leaflet/Leaflet/blob/master/src/layer/tile/TileLayer.WMS.js

**Comment by cmulders** [comment]
The gh-pages is the better starting point, more documentation and some unit tests. However, some tests are silly, so maybe skip some of those.. Still learning :) 

I like the options specific for L.TileLayer.WMS, `wmsparams`. Zoomify is a fairly straightforward format, only the tiles are split between dirs and have funny dimensions. So mostly the code is correcting these inconsistencies with world-sized tile layers and does not have a lot of overlap with WMS I think.

I have not tested with rc1, but it is possibly already compatible, at least 0.7 function calls have been replaced with 1.0-beta calls! I check up with the code soon, possibly polishing it up and make it 1.0 plugin worthy. Would be great if you have any suggestions :+1:

**Comment by hyperknot** [comment]
@cmulders if you come up with a candidate version for Leaflet rc1, I'd be happy to review it and/or help with the development.

**Comment by GeneGIGA** [comment]
I've been working with hyperknot and we have a working version with rc1 and can likely put something back up for that.
